exasol_transformers_extension/utils/model_specification.py

```diff
@@ -1,25 +1,73 @@
 from pathlib import PurePosixPath, Path
+import transformers
+
 
 class ModelSpecification:
     """
     Class describing a model.
     """
-    def __init__(self, model_name: str):
+    def __init__(self, model_name: str, task_type: str):
         # task_type, model_version
         self.model_name = model_name
+        self.task_type = task_type
+
+    def set_task_type_from_udf_name(self, text):
```

Comment:
is this actually used, and if we need it, should not the UDF simply create the ModelSpec with the correct task_type string, instead we need another translation.